### PR TITLE
[operator] Change runbook_url to an actual url in the PrometheusRule

### PR DESCRIPTION
**Pull request by bogdan-at-adobe**
This depends on https://github.com/open-telemetry/opentelemetry-operator/pull/3240

### CODE REVIEW

File: charts/opentelemetry-operator/templates/prometheusrule.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ spec:
         {{- end }}
       annotations:
         description: '{{`Reconciliation errors for {{ $labels.controller }} is increasing and has now reached {{ humanize $value }} `}}'
-        runbook_url: 'Check manager logs for reasons why this might happen'
+        runbook_url: 'https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/runbooks.md#reconcileerrors'
```
Comment (JaredTan95) [inline charts/opentelemetry-operator/templates/prometheusrule.yaml:35]: Can we make this configurable in `values.yaml`?

Comment (bogdan-at-adobe) [reply charts/opentelemetry-operator/templates/prometheusrule.yaml]: yes, done.

File: charts/opentelemetry-operator/values.yaml [hunk 2]
```diff
@@ -41,7 +41,7 @@ manager:
     repository: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
     tag: ""
   collectorImage:
-    repository: ""
+    repository: "otel/opentelemetry-collector"
```
Comment (JaredTan95) [inline charts/opentelemetry-operator/values.yaml:44]: pls revert this line

Comment (bogdan-at-adobe) [reply charts/opentelemetry-operator/values.yaml]: uh, yeah, my bad, done.

**Comment by bogdan-at-adobe** [comment]
Moved the runbook PR to the opentelemetry.io repo with [this PR](https://github.com/open-telemetry/opentelemetry.io/pull/5131). Ill wait for that one and update the urls here after that one is merged.

**Review by Allex1 — APPROVED** [review]

**Review by jaronoff97 — APPROVED** [review]

**Comment by jaronoff97** [comment]
@bogdan-at-adobe thank you for your contributions here and in the opentelemetry.io repo. I appreciate your work and patience :)

**Pull request merged**